staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -690,9 +687,8 @@ func (meal *cfgMeal) digestNewPLsLocked(newPLs []*flowcontrol.PriorityLevelConfi
 			klog.V(3).Infof("Priority level %q was undesired and has become desired again", pl.Name)
 			state.quiescing = false
 		}
-		if state.pl.Spec.Limited != nil {
-			meal.shareSum += float64(state.pl.Spec.Limited.NominalConcurrencyShares)
-		}
+		nominalConcurrencyShares, _, _ := plSpecCommons(state.pl)
```

Comment:
nit: what I was originally asking for is to split the refactorings (e.g. things like introducing plSpecCommons function or changed apf_controller_debug.go) which are easy to review and makes the main PR smaller [and thus easier to review]. But I guess it might be too late now...

Reply:
I am not sure I understand.  The `plSpecCommons` function is a small part of this PR.  This PR is much smaller than #118782 .

Reply:
plSpecCommons + all plumbing of it + apf_controler_debug, etc. are maybe not majority, but non-negligible part
and these are no-brainer and trivial to reason about
given that review is exponential to the size of PR, even if that's 1/3rd of PR, the impact on review time is actually quite significant :)

Reply:
That said - as I mentioned, it's probably too late.
I managed to go over it and it looks fine now.